Exercises/chapter3/e1-looping-a-triangle.js

```diff
@@ -0,0 +1,5 @@
+let size = 8;
+let output = "";
+for (let i = 0; i < size; i++) {
+  console.log(output += "#");
```

Comment:
This is a little harder to read than it needs to be since it forces the reader to think about the result of the +=. I'd recommend splitting it into two lines.